### PR TITLE
Fix type long bug

### PR DESCRIPTION
**Pull request by marsishandsome**

### CODE REVIEW

**Comment by marsishandsome** [comment]
@zhexuany PTAL

File: core/src/main/scala/com/pingcap/tispark/utils/TiUtil.scala [hunk 1]
```diff
@@ -114,28 +114,50 @@ object TiUtil {
...
       case _: IntegerType =>
         if (tp.asInstanceOf[IntegerType].isUnsignedLong) {
           DataTypes.createDecimalType(20, 0)
         } else {
-          sql.types.LongType
+          if (version == 0) {
+            sql.types.LongType
+          } else {
+            tp.getType match {
+              case MySQLType.TypeLong     => sql.types.IntegerType
```
Comment (birdstorm) [inline core/src/main/scala/com/pingcap/tispark/utils/TiUtil.scala]: we might need to consider Unsigned Integer here.

**Comment by ilovesoup** [comment]
Please provide corresponding issue / Jira number.

**Comment by ilovesoup** [comment]
And please don't put user name on any description.

**Pull request closed without merging**